packages/components/src/components/SquiggleViewer/SquiggleValueResultViewer.tsx

```diff
@@ -1,12 +1,12 @@
 import { memo } from "react";
 
-import { result, SqError, SqValue } from "@quri/squiggle-lang";
+import { result, SqErrorList, SqValue } from "@quri/squiggle-lang";
 
 import { valueHasContext } from "../../lib/utility.js";
 import { PlaygroundSettings } from "../PlaygroundSettings.js";
 import { SquiggleValueChart } from "./SquiggleValueChart.js";
 
-export type SqValueResult = result<SqValue, SqError>;
+export type SqValueResult = result<SqValue, SqErrorList>;
```

Comment:
`SqErrorList` here and in other files is still unused, all lists have a single error. But this will be useful in the future.

Reply:
Happy to see steps like this